gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>I'd recommend removing the larger block and just having a bullet point for </p><p> </p><p><code>resourcemanager.projects.setIamPolicy</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Not sure why the formatting is messed up. Added as bullet now</p>